reputation/reputation.go

```diff
@@ -2,12 +2,18 @@ package reputation
 
```

Comment:
This package now leverage the work of the created indexes.

Reply:
Slick